department-of-veterans-affairs/caseflow
# Case Review & BVA Intake Model Cleanup

Completes #13865, Cleans up #13908

### Description
Removes the special cleanup code that fixed production data for the Case Review team.

 - [x] Not to be deployed until production database shows the correct Case Review type, `CaseReview`.

### Acceptance Criteria
- [ ] Code is clean
- [ ] Seeds create dev user & admin for the org

### Testing Plan

Check Prod:
  1. If the CaseReview Organization type is
    - `BvaIntake`, do not merge this PR.
    - `CaseReview`, good to go!

In Dev:
  1.  Checkout master and fully reseed your database make reset
  1. Checkout this branch and invoke the singletons in the rails console. No errors.

```ruby
> CaseReview.singleton
> BvaIntake.singleton
```

 1. Reseed with this branch checked out, confirm the new BVA test users are available to select at `localhost:3000/test/users`

### User Facing Changes
None
